lib/ssg/downloader.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'net/http'
+
+module Ssg
+  # Downloads SCAP datastreams from the SCAP Security Guide
+  # https://github.com/ComplianceAsCode/content
+  class Downloader
+    RELEASES_API = 'https://api.github.com/repos'\
```

Comment:
How official source is this?

Reply:
It doesn't look good :( (from #openscap on irc.devel.rh):

```
13:00   akofink ╡ o/ Hi - how reliably do the releases at https://github.com/ComplianceAsCode/content/releases match what is packaged and shipped as RPMs to RHEL?
13:01         ⤷ ╡ like, RHEL7 has scap-security-guide-0.1.43-13.el7.noarch - are the datastreams provided guaranteed to match tag v0.1.43 on github?
13:11   ascheel ╡ akofink: (I don't work on this team any more, and everyone who still does is UTC+2) -- I think dist-git is your friend here.
13:13   ascheel ╡ akofink: `rhpkg clone scap-security-guide` -- 7.7 has what you see there, 7.8 has a rebase to v0.1.46. There's a number of patches (~28) which have been picked on top of 0.1.43 tarball.
13:14   akofink ╡ hrmm, okay :/ it seems more complicated than I expected
13:14   ascheel ╡ akofink: What are you looking for in particular while I have it up?
13:15   akofink ╡ I'm writing a thing to download and extract certain datastreams of specific versions of the SSG, and we'd like to sync the SSG provided to RHEL6, 7, and 8. It seems like the only way to really do that is to look at the RHEL package repos
13:16         ⤷ ╡ https://github.com/dLobatog/openscap_parser/pull/12 if you're interested
13:16         ⤷ ╡ (the openscap ffi gem has memory leaks last we checked)
13:17   ascheel ╡ Yeah, sorry. The RHEL packaging process kinda hinders SSG here.
13:17   akofink ╡ thanks for the pointers though :)
13:18   ascheel ╡ akofink: Yeah looks like they've picked ~66 commits on top of that tag.
13:18   akofink ╡ geez
```